addons/website/models/website_controller_page.py

```diff
@@ -36,6 +39,10 @@ class WebsiteControllerPage(models.Model):
         default="grid",
     )
 
+    _sql_constraints = [
+        ('unique_name_slugified', 'UNIQUE(page_type, name_slugified)', 'url should be unique')
+    ]
```

Comment:
I thought `_sql_constraints` had to be grouped with the other `_*` attributes... but I might be wrong since it seems this location is more common in the codebase.